crates/ruma-federation-api/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Improvements:
 
 * Add master_keys and self_signing keys to keys::get_keys::v1 response
+* Improve types of keys::claim_keys::v1 response
```

Comment:
"Improve types" is too vague for a changelog entry. Also, this is a breaking change.

Reply:
I've changed the line